### PR TITLE
simple CPU bound pipelines

### PR DESCRIPTION
**Pull request by drewrip**
Not sure where the best place was for these so I added a new dir for non-notebook pipelines. These are just some simple workloads to generate CPU load. I don't believe Trimaran takes into account memory usage but I am working on a simple memory bound pipeline for later.

### CODE REVIEW

**Comment by kpouget** [comment]
@drewrip , can we try something? 
* put the files in https://github.com/openshift-psap/ci-artifacts/tree/main/testing/pipelines/notebooks
* add a notebook based on this one https://github.com/openshift-psap/ci-artifacts/blob/main/testing/pipelines/notebooks/hello-world/kfp_hello_world.ipynb (there shouldn't be much to adapt)
* then I'll push a commit to pass it to this entrypoint, instead of the hard coded hello-world https://github.com/openshift-psap/ci-artifacts/blob/main/toolbox/pipelines.py#L29

**Comment by drewrip** [comment]
This should add the accompanying notebooks!

File: testing/pipelines/command_args.yaml [hunk 1]
```diff
@@ -63,6 +63,7 @@ driver/cluster set_project_annotation/test_toleration:
 pipelines run_kfp_notebook:
   namespace: {{ rhods.pipelines.namespace }}
   dsp_application_name: {{ rhods.pipelines.application.name }}
+  notebook: {{tests.pipelines.notebook}}
```
[the remark this answers is not in the document]
Comment (drewrip) [reply testing/pipelines/command_args.yaml]: Fixed with most recent commit. Squashed into https://github.com/openshift-psap/ci-artifacts/pull/748/commits/430246855c3c0bb6d9de41d8a8c9103b4413d0a7

[the remark this answers is not in the document]
Comment (kpouget) [reply testing/pipelines/command_args.yaml]: > [pod/ci-artifacts-0-n8k6j/main] ERROR:root:from_config failed: run_kfp_notebook() got an unexpected keyword argument 'notebook'

[from this log file](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/748/pull-ci-openshift-psap-ci-artifacts-main-pipelines-light/1666807507890212864/artifacts/light/test/artifacts/000__pipelines_run_many/000__local_ci__run_multi/ci-pods_logs/ci-artifacts-0-n8k6j.log)
this should match the parameter name:
```
notebook_filename: {{ tests.pipelines.notebook }}
```

**Comment by kpouget** [comment]
`pipeline_hash` [test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/748/pull-ci-openshift-psap-ci-artifacts-main-pipelines-light/1666899339294806016/artifacts/light/test/artifacts/002__plots/report_00_report:_error_report.html) :)

**Comment by kpouget** [comment]
`pipeline_pi` [test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/748/pull-ci-openshift-psap-ci-artifacts-main-pipelines-light/1667046701384339456/artifacts/light/test/artifacts/002__plots/report_00_report:_error_report.html)

**Comment by kpouget** [comment]
pipeline_pi [test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/748/pull-ci-openshift-psap-ci-artifacts-main-pipelines-light/1667046701384339456/artifacts/light/test/artifacts/002__plots/report_00_report:_error_report.html)
pipeline_hash [test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/748/pull-ci-openshift-psap-ci-artifacts-main-pipelines-light/1666899339294806016/artifacts/light/test/artifacts/002__plots/report_00_report:_error_report.html) 
merging :)

**Pull request merged**